variables.tf

```diff
@@ -6,6 +6,11 @@ variable "topic_name" {
   description = "A prefix that will be added to resources so that they are unique."
 }
 
+variable "random_postfix" {
+  description = "Boolean for adding a random string to the end of the iam role, iam policy, topic and function names"
+  default     = "false"
```

Comment:
I think we should keep this as `true` since it's the current behaviour. Otherwise this change will break backwards compatibility.

Reply:
Terraform noob question: 

Does Terraform support boolean variables? Could this variable be defined as `default = true` rather than `default = "true"`.  If so, I was thinking we could simplify the ternary expression as well.

(BTW: Hi. I'm John. I work with LH in privacy services.)

Reply:
Yes, it does, but! The ternary operation should be rewriten from `${var.random_postfix == "true" ? ` to `${var.random_postfix ? ` because there was some issue in comparing boolean values with variables (bizzare isn't it), I am not sure if it got fixed.

Reply:
Related issue https://github.com/hashicorp/terraform/issues/13389

Reply:
Cool, thanks.  Yes, that's what I intended by "simplify the ternary". :)  It is no doubt a form of bias from being a longtime rubyist, but the latter syntax reads more expressively to me.  (Especially if we make the variable name something like `has_random_postfix` or similar.)

Reply:
Thanks for your comments! I have changed the bool to be `true` by default and simplified the ternary expression as you guys suggested. I tested out `${var.random_postfix == true ?` earlier and that didn't work, which is why I used the `"true"`. Don't know why I didn't think of using just `${var.random_postfix ?`, but its anyway been fixed now.

Reply:
If you guys approve of the PR now, then someone with write access to this repo have to merge :)